src/ZoneServer/Buffs/Handlers/Agility_Buff.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Melia.Shared.Tos.Const;
+using Melia.Zone.Buffs.Base;
+using Melia.Zone.Network;
+using Melia.Zone.World.Actors.Characters;
+
+namespace Melia.Zone.Buffs.Handlers
+{
+	/// <summary>
+	/// Handle for the Agility Buff, which increases the target's movement speed.
+	/// </summary>
+	[BuffHandler(BuffId.Agility_Buff)]
+	public class Agility_Buff : BuffHandler
+	{
+		private int speedBonus = 0;
+
+		public override void OnStart(Buff buff)
+		{
+			var target = buff.Target as Character;
+
+			// It is not applyed to characters without Shoes
+			if (target != null && target.Inventory.GetEquip(EquipSlot.Shoes) != null)
+			{
+				// It is not applyed to characters without Gloves
+
+				if (target != null && target.Inventory.GetEquip(EquipSlot.Gloves) != null)
+				{
+					if (buff.Data.Level == 1 || buff.Data.Level == 2)
+					{
+						speedBonus = 2;
+					}
+					else if (buff.Data.Level == 3 || buff.Data.Level == 4)
+					{
+						speedBonus = 3;
```

Comment:
It seems like this code is incorrect. The skill description mentions a bonus of 1 for levels 1\~3, 2 for 4~6, etc. Am I missing something? This structure could also be simplified a little to make it more concise. What do you think?
```cs
speedBonus = (int)Math.Ceiling(level / 3f);
```